### PR TITLE
fix:  add pre-filled option on adding an option to a checkbox or radio button

### LINKED ISSUE #8503: [Bug]: Checkbox group widget appears blank when more than 4 Options are added
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

[LOOM DEMO](https://www.loom.com/share/eff3a122318d481ea2ddb61c3df3a64c)

### Steps To Reproduce

```markdown
1. Drag and drop a checkbox group widget
2. Open property pane
3. Click on '+ Options' 
4. Observe when more than 4 Options are added the widget appears blank
```

### Environment

Release

### Version

Cloud

### PR DESCRIPTION
**Pull request by vicky-primathon**
## Description

Fixes #8503  (issue)

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:red_circle: Total coverage has decreased</summary>

    // Code coverage diff between base branch:release and head branch: fix/add-option 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :red_circle: | total | 55.01 **(0)** | 36.5 **(0)** | 34.72 **(-0.01)** | 55.4 **(-0.01)**
 :green_circle: | app/client/src/components/propertyControls/KeyValueComponent.tsx | 15.79 **(0.08)** | 24 **(0)** | 0 **(0)** | 17.14 **(-0.05)**
 :red_circle: | app/client/src/utils/autocomplete/TernServer.ts | 52.71 **(-0.23)** | 40.83 **(-0.84)** | 36.21 **(0)** | 56.74 **(-0.25)**</details>

### CODE REVIEW

File: app/client/src/components/propertyControls/KeyValueComponent.tsx [hunk 1]
```diff
@@ -148,9 +149,28 @@ export function KeyValueComponent(props: KeyValueComponentProps) {
   function addPair() {
     let { pairs } = props;
     pairs = Array.isArray(pairs) ? pairs.slice() : [];
-    pairs.push({ label: "", value: "" });
+    pairs.push({
```
Comment (SatishGandham) [inline app/client/src/components/propertyControls/KeyValueComponent.tsx:152]: What are pairs and renderPairs?

**Review by SatishGandham — COMMENTED** [review]
Not related to this current issue.
Since the options are unique, can we not use them as key instead of generating it?

**Review by SatishGandham — APPROVED** [review]

**Comment by shwetha-ramesh** [comment]
@vicky-primathon tested this PR. DP looks good. Checkbox group and Radiogroup widget now has pre-filled name and value with prefix added.
Moving to Done.

**Pull request merged**